docs/UserGuide.md

```diff
@@ -1,21 +1,16 @@
----
-layout: page
-title: User Guide
----
-
 EventfulNUS is a **desktop app for managing contacts and events specifically for the Inter-Faculty Games hosted annually
 at the National University of Singapore. While optimised for use via a Command Line Interface** (CLI), it also has the benefits of a Graphical User Interface (GUI). If you can type fast, you will certainly benefit from event organisation tasks being done faster than traditional GUI apps.
 
 ---------------------------------------------------------------------------------------------------------------------
 ## Table of Contents
 1. [Quick start](#quick-start)
 2. [Features](#features)
-3. [Person commands](#persons)
-4. [Event commands](#events)
-5. [More features](#more-features)
-6. [FAQ](#faq)
-7. [Known issues](#known-issues)
-8. [Command summary](#command-summary)
+   * [Person commands](#persons)
+   * [Event commands](#events)
+   * [Miscellaneous features](#miscellaneous-features)
+3. [FAQ](#faq)
+4. [Known issues](#known-issues)
+5. [Command summary](#command-summary)
 --------------------------------------------------------------------------------------------------------------------
 ## Quick start
 
```

Comment:
Maybe we want to rewrite the instructions for installation to be more non-tech friendly?

Reply:
not sure how